extensions/ql-vscode/package.json

```diff
@@ -61,6 +61,8 @@
     "onCommand:codeQL.chooseDatabaseLgtm",
     "onCommand:codeQLDatabases.chooseDatabase",
     "onCommand:codeQLDatabases.setCurrentDatabase",
+    "onCommand:codeQLDatabasesExperimental.openConfigFile",
+    "onCommand:codeQLDatabasesExperimental.setSelectedItem",
```

Comment:
From https://code.visualstudio.com/api/extension-guides/command#creating-a-user-facing-command, we need to add any user-facing command to the extension's `activationEvents`